src/bun.js/modules/NodeBufferModule.h

```diff
@@ -160,8 +176,15 @@ DEFINE_NATIVE_MODULE(NodeBuffer) {
   put(JSC::Identifier::fromString(vm, "File"_s),
       globalObject->JSDOMFileConstructor());
 
-  put(JSC::Identifier::fromString(vm, "INSPECT_MAX_BYTES"_s),
-      JSC::jsNumber(50));
+  {
+    PropertyDescriptor descriptor;
+    descriptor.setConfigurable(true);
+    descriptor.setEnumerable(true);
+    descriptor.setWritable(false);
+    descriptor.setGetter(JSFunction::create(vm, lexicalGlobalObject, 0, String("get"_s), jsFunction_INSPECT_MAX_BYTES_getter, {}, {}, {}, {}));
```

Comment:
Please use custom getter/setters for these. JSFunctions are slower and much more code than they need to be

Reply:
would that match https://github.com/nodejs/node/blob/v22.9.0/lib/buffer.js#L1344-L1353 ?

Reply:
yes